### ISSUE TITLE
Peer dependencies are out of sync

### ISSUE THREAD

**Issue by Methuselah96** [DevExpress/devextreme-reactive#1362]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...
- [x] React Grid
- [ ] React Chart
- [ ] Vue Grid

### Current Behaviour
`dx-react-grid@1.7.0` and `dx-react-grid-material-ui@1.7.0` have peer dependencies on `dx-core@1.6.0` and `dx-react-core@1.6.0` and produce a warning when installing the latest version of the packages.

### Expected Behaviour
They should support the packages at the latest versions as peer dependencies without warnings.

### Steps to Reproduce (for Bugs)
Install `dx-react-grid`, `dx-react-grid-material-ui`, `dx-core`, and `dx-react-core` at version 1.7.0 using Yarn (or maybe npm) and observe the warnings.

### Environment
- devextreme-reactive: none
- react: none
- vue: none
- browser: none
- bootstrap: none
- react-bootstrap: none
- material-ui: none

**Comment by franklixuefei** [comment]
```
npm WARN @devexpress/dx-react-grid-material-ui@1.7.0 requires a peer of @devexpress/dx-grid-core@1.6.0 but none is installed. You must install peer dependencies yourself.
npm WARN @devexpress/dx-react-grid-material-ui@1.7.0 requires a peer of @devexpress/dx-react-core@1.6.0
but none is installed. You must install peer dependencies yourself.
npm WARN @devexpress/dx-react-grid-material-ui@1.7.0 requires a peer of @devexpress/dx-react-grid@1.6.0
but none is installed. You must install peer dependencies yourself.
npm WARN @devexpress/dx-grid-core@1.7.0 requires a peer of @devexpress/dx-core@1.6.0 but none is installed. You must install peer dependencies yourself.
```

**Comment by franklixuefei** [comment]
Also, consider supporting `@material-ui/core` and `@material-ui/icons` of version `3.0.0+` :-D

**Comment by MaximKudriavtsev** [comment]
Hi,
 
We fixed this issue in [v1.7.2](https://github.com/DevExpress/devextreme-reactive/releases/tag/v1.7.2). Please update your version of the React Grid.
